src/lib/ExtensionPreviewURL.ts

```diff
@@ -49,6 +50,14 @@ export default class ExtensionPreviewURL {
 
             // Read extension context file
             const extensionContext = new ExtensionContext();
+            // Validate environment before extension preview
+            const contextEnv = extensionContext.get(CONSTANTS.EXTENSION_CONTEXT.CURRENT_ENV) || 'api.fynd.com';
```

Comment:
@jinalviranii Do we require default value here ?
Ideally it should be env that user has logged in

Reply:
Kept default value as logged in user env value